packages/snaps-simulator/.eslintrc.js

```diff
@@ -0,0 +1,77 @@
+module.exports = {
+  root: true,
+
+  extends: ['../../.eslintrc.js'],
+
+  parserOptions: {
+    tsconfigRootDir: __dirname,
+  },
+
+  settings: {
+    'import/extensions': ['.ts', '.tsx'],
+  },
+
+  overrides: [
+    {
+      files: ['*.ts', '*.tsx'],
+      extends: [
+        'plugin:react/recommended',
+        'plugin:react/jsx-runtime',
+        'plugin:react-hooks/recommended',
+      ],
+      rules: {
+        '@typescript-eslint/no-shadow': [
+          'error',
+          {
+            allow: ['Text'],
+          },
+        ],
+
+        'react/display-name': 'off',
+        'react/prop-types': 'off',
+
+        // TODO: Investigate why this is needed.
+        'node/no-unpublished-import': 'off',
```

Comment:
Is this only a problem in the Webpack config? If so, maybe we just disable for that one file?

Reply:
Which files?

Reply:
```
./packages/snaps-simulator/src/features/manifest/sagas.test.ts
  1:28  error  "redux-saga-test-plan" is not published  node/no-unpublished-import

./packages/snaps-simulator/src/features/polling/sagas.test.ts
  1:23  error  "jest-fetch-mock" is not published       node/no-unpublished-import
  2:28  error  "redux-saga-test-plan" is not published  node/no-unpublished-import

./packages/snaps-simulator/src/features/simulation/hooks.test.ts
  3:28  error  "redux-saga-test-plan" is not published  node/no-unpublished-import

./packages/snaps-simulator/src/features/simulation/sagas.test.ts
  4:28  error  "redux-saga-test-plan" is not published  node/no-unpublished-import

./packages/snaps-simulator/webpack.config.ts
   1:32  error  "@pmmmwh/react-refresh-webpack-plugin" is not published  node/no-unpublished-import
   2:35  error  "favicons-webpack-plugin" is not published               node/no-unpublished-import
   3:31  error  "html-webpack-plugin" is not published                   node/no-unpublished-import
   4:39  error  "monaco-editor-webpack-plugin" is not published          node/no-unpublished-import
```